conda_smithy/feedstock_io.py

```diff
@@ -3,9 +3,18 @@
 import os
 import shutil
 import stat
-
-
-def get_repo(path, search_parent_directories=True):
+from io import TextIOWrapper
```

Comment:
Please check this import

Reply:
now I saw it. Mh... I am unsure

Reply:
this was a suggestion when I ran [monkeytype](https://github.com/Instagram/MonkeyType), but if it doesn't make sense we can take it out..